enhancements/testing/openshift-tests-extension.md

```diff
@@ -0,0 +1,668 @@
+---
+title: extended-platform-tests
+authors:
+  - "@jupierce"
+  - "@stbenjam"
+reviewers:
+  - "@deads2k"
+creation-date: 2024-09-05
+last-updated: 2024-09-05
+status: implementable
+---
+
+# OpenShift Tests Extensions
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Define a pattern by which repos other than origin can contribute tests to 
+OpenShift test suites.
+// Layered on commits from Stephen with good description from gdoc if we move forward.
+
+## Motivation
+
+Increase the number of tests in the OpenShift Tests Suite by reducing the
+effort, complexity, and risk of introducing new tests.
+
+### Goals
+
+1. Reduce the effort, complexity, and risk of introducing new tests to OpenShift.
+2. Allow tests to be introduced in the same pull request as the PR which fixes a bug or introduces a feature.
+3. Allow component owners to introduce new test suites / add tests to existing suites.
+4. Allow non-OpenShift components, including layered products, to participate in extending OpenShift tests and Component Readiness.
+5. Maintain and increase testing coverage.
+6. Maintain and increase code quality.
+7. Maintain and improve the signal TRT provides through Component Readiness.
+8. Provide a standard model for testing component configurations.
+9. Allow distributed contribution of tests while introducing centralized mechanisms to require important tests from component owners and improve coverage over time.
+10. Improve testing efficiency by reducing initialization time.
+
+### Non-Goals
+1. Fully decompose `openshift-tests` or decentralize test orchestration & reporting.

```

Comment:
Might be worth noting here this is purely for e2e ginko tests, and "monitortests" which are instantiated at the start, watch the cluster, and generate junit results at the end based on what it observed, will remain in origin.

Reply:
@dgoodwin - I've added "list monitors" and "run-monitor" if you want to take a look.